### PR TITLE
[WIP] Convert play-2.4 instrumentation to instrumenter api

### PR DESCRIPTION
**Pull request by shivanshu1333**
Part of https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/2713

### CODE REVIEW

**Comment by shivanshu1333** [comment]
Note to reviewers: I'm a first-time contributor to opentelemetry-java-instrumentation, I'm still understanding how to do the migration from play-2.4 instrumentation to instrumentation API from the various previous migration example.

File: instrumentation/play/play-2.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/play/v2_4/ActionInstrumentation.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import static io.opentelemetry.javaagent.extension.matcher.AgentElementMatchers.hasClassesNamed;
 import static io.opentelemetry.javaagent.extension.matcher.AgentElementMatchers.implementsInterface;
 import static io.opentelemetry.javaagent.instrumentation.play.v2_4.PlayTracer.tracer;
+import static io.opentelemetry.javaagent.instrumentation.play.v2_4.PlaySingletons.instrumentor;
```
Comment (trask) [inline instrumentation/play/play-2.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/play/v2_4/ActionInstrumentation.java:11]: hi @shivanshu1333! it looks like `PlaySingletons` is missing from the PR

Comment (iNikem) [reply instrumentation/play/play-2.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/play/v2_4/ActionInstrumentation.java:11]: @shivanshu1333 can add this file please?

Comment (trask) [reply instrumentation/play/play-2.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/play/v2_4/ActionInstrumentation.java:11]: @shivanshu1333 would you object if someone else picks up this PR from where you left off? We would like to close out the last Instrumenter API conversions soon. Thx!

**Comment by trask** [comment]
this has been implemented now in #4853

**Pull request closed without merging**